website/docs/language/tests/index.mdx

```diff
@@ -1009,4 +1010,60 @@ Run D (parallel: true)
 Run E (parallel: true)
 ```
 
-Runs A and B execute simultaneously. Run C waits until both A and B are done before starting. Finally, runs D and E run in parallel, but only after C completes.
\ No newline at end of file
+Runs A and B execute simultaneously. Run C waits until both A and B are done before starting. Finally, runs D and E run in parallel, but only after C completes.
+
+## Backends
+
+By default, Terraform tests start with an empty state and provisions all resources defined in the configuration.
+However, if a `backend` block is used in a test file then the test can start with a non-empty state file.
+This allows tests to reuse long-lived pieces of infrastructure and can be used to reduce the duration of tests
+that include resources that are slow to provision, such as Kubernetes clusters.
+
+A `backend` block can be added once in a test file, to the first run block that represents an `apply` operation:
```

Comment:
It can be added once in a test file, for each state key. Can we reframe this to make that clearer, or perhaps add that information in another paragraph? If the file uses multiple state keys, then there can be more than one block

Reply:
To tackle this I've updated the draft to have an initial simple example, and then another example showing the effects of module state/state key on use of a backend block.